precompiles/staking/StakingI.sol

```diff
@@ -142,6 +148,21 @@ interface StakingI is authorization.AuthorizationI {
         uint256 value
     ) external returns (bool success);
 
+    /// @dev Defines a method for edit a validator.
+    /// @param description Description parameter to be updated, an string "[do-not-modify]"
+    /// means keeping it as is and not updating it.
```

Comment:
Hey @luchenqun, I'm very happy to see you around again! Thanks for adding the fixes, howerver, this description is still not very clear. I would suggest to write:

```suggestion
    /// @param description Description parameter to be updated. Use the string "[do-not-modify]" as the value of 
    /// fields that should not be updated.
```